packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,934 @@
+selection, the <Keyboard>Ctrl</Keyboard> key on Windows or the <Keyboard>Option</Keyboard> key on macOS can be held while pressing the arrow keys. Holding this modifier while pressing
+the <Keyboard>Space</Keyboard> key toggles selection for the focused row, which allows multiple selection of non-contiguous items. On touch screen devices, selection always behaves as toggle
+since modifier keys may not be available. This behavior emulates native platforms such as macOS and Windows.
+
+```tsx example
+// Using the same list as above
+<PokemonList selectionStyle="highlight" aria-label="Highlight selection list" />
+```
+
+### Row actions
+
+ListView may be used in use cases where users can perform actions on rows, such as navigating into items to open them or get more details. The `onAction` prop can be used to enable
+these row actions, but the interaction to trigger the row action depends on the selection style and current selection state of the ListView. When `onAction` is provided to a checkbox selection style ListView,
+clicking the row with a mouse or pressing the <Keyboard>Enter</Keyboard> key will trigger onAction. Row selection is performed by pressing on the row's checkbox itself or by pressing <Keyboard>Space</Keyboard>.
+While any number of rows are selected, subsequent clicks on a row will toggle selection instead. During this state `onAction` can only be triggered via the <Keyboard>Enter</Keyboard> key, but deselecting all selected
+rows will restore the original "click for onAction" behavior. If `onAction` is provided to a highlight selection ListView, double clicking the row triggers `onAction` and a single click selects the row instead.
+
+On touch devices, `onAction` is always the primary tap interaction, and tapping on the row's checkbox or long pressing the row will select the row. In highlight selection, a long press will also shift the ListView into selection mode,
+which displays checkboxes to perform selection. While the ListView is in this state, tapping the row will toggle selection instead of triggering `onAction`. Deselecting all items exits selection mode and hides the checkboxes.
+Double clicking matches the behavior of desktop platforms like macOS and Windows, and a separate selection mode matches mobile platforms like iOS and Android.
+
+```tsx example
+// Checkbox selection with onAction
+<Flex wrap gap="size-300">
+  <PokemonList onAction={key => alert(`Opening item ${key}...`)} aria-label="Checkbox selection list with row actions" width="size-2400" />
+  <PokemonList selectionStyle="highlight" onAction={key => alert(`Opening item ${key}...`)} aria-label="Highlight selection list with row actions" width="size-2400" />
+</Flex>
+```
+
+## Drag and drop
+
+To enable drag and drop in a ListView, you must provide the respective drag and drop hooks sourced from `useDragHooks` and `useDropHooks` to the ListView's `dragHooks` and `dropHooks` props respectively.
```

Comment:
The examples may error/misbehave when trying to perform a second bulk drag, fixed by a different PR